spec/features/availability_spec.rb

```diff
@@ -18,6 +18,25 @@
     end
   end
 
+  # This test is quite brittle and will break once this item is removed from
+  # CDL. It seems valid to remove it if it hasn't been refactored by then, and
+  # at least create an issue to replace it with a jest test during a refactor of
+  # this javascript.
```

Comment:
@hackmastera can you open an issue and log this test?

Reply:
It's part of #1094 and #823 which I'm currently working on. I'd like to see where I am at the end of the work cycle.